### PR TITLE
Better dupes

### PR DESCRIPTION
**Pull request by andrewconner**

### CODE REVIEW

**Comment by andrewconner** [comment]
So far, here are the patterns I have:

```
^https*://www\.facebook\.com/login.*$
^https*://.*.google.com.*/ServiceLogin.*$
^https*://app.asana.com.*$
^https*://www.linkedin.com/uas/login.*$
```

File: shoebox/app/com/keepit/model/Unscrapable.scala [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  import DBSession._
+  import scala.util.matching.Regex
+
+  override lazy val table = new RepoTable[Unscrapable](db, "unscrapable") {
+    def pattern = column[String]("pattern", O.NotNull)
+    def state = column[State[Unscrapable]]("state", O.NotNull)
+    def * = id.? ~ createdAt ~ updatedAt ~ pattern ~ state <> (Unscrapable, Unscrapable.unapply _)
+  }
+
+  def allActive()(implicit session: RSession): Seq[Unscrapable] =
```
Comment (eishay) [inline shoebox/app/com/keepit/model/Unscrapable.scala:59]: looks like allActive return value could be safely cached with ttl or just as lazy val, right?

File: shoebox/conf/evolutions/shoebox/27.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# --- !Ups
+
+CREATE TABLE unscrapable (
+    id bigint(20) NOT NULL AUTO_INCREMENT,
+    pattern varchar(256) NOT NULL,
+    state varchar(20) NOT NULL,
```
Comment (eishay) [inline shoebox/conf/evolutions/shoebox/27.sql:6]: index on state?
I know its a super fast query with a small table, but as a rule of thumb we shouldn't have queries without an appropriate index

Comment (andrewconner) [inline shoebox/conf/evolutions/shoebox/27.sql:6]: Added

File: shoebox/app/com/keepit/model/Unscrapable.scala (hunk 1 above)
Comment (andrewconner) [inline shoebox/app/com/keepit/model/Unscrapable.scala:59]: As per our conversation, leaving it out for now, and will use the new caching interface when it's ready.

**Pull request merged**